### ISSUE TITLE
Support convergence

### ISSUE THREAD

**Issue by pharaone** [pixelr0/balls#2]
Please support convergence (i386 and amd64) packaging balls in snap

**Comment by pixelr0** [comment]
right, there should be a way to make a fat click package, investigating

**Comment by pharaone** [comment]
I tell you to create a snap package because the final version of Ubuntu 16.04 (personal unity8 / mir) .Click packages will not work (even if you do not work even now) to leave only to .snap space

**Comment by pharaone** [comment]
Now?

**Comment by pharaone** [comment]
Now?

**Comment by pixelr0** [comment]
still nope

**Comment by pharaone** [comment]
you can do it after the release of the final version?

**Comment by pixelr0** [comment]
so.. Ubuntu SDK doesn't support fat clicks, nor snaps so it's a pain to manually make them. i'm waiting for support in SDK first

**Comment by pixelr0** [comment]
ok, so now you can install snaps on 16.04 :D i'm reading the documentation. will try to make a snap, lets see

**Comment by pharaone** [comment]
if I'm not mistaken you should do it using snapcraft

**Comment by pharaone** [comment]
Now you can?
